### PR TITLE
ROX-17673: Prepare target for sensor-integration tests

### PR DESCRIPTION
**Pull request by vikin91**
This PR is meant to prepare a dummy (always passing) test target from in https://github.com/stackrox/stackrox/pull/6399

The goal is to move the sensor integration tests away from the nongroovy step as the integration tests are now larger and keep growing. This would allow us (who work on Sensor) to iterate quicker when adding new features and tests in the upcoming epic. This will happen in https://issues.redhat.com/browse/ROX-17674.

### CODE REVIEW

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-10.yaml [hunk 1]
```diff
@@ -42,6 +42,16 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-aws
   timeout: 5h0m0s
+- as: ocp-sensor-integration-tests
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-10.yaml:45]: openshift-ci will prefix the job with the variant name (ocp-4.10 in this case) so there is no need for `ocp-` in the `as:` values throughout. When your stackrox/stackrox PR merges they will show up as `ocp-4-*-sensor-integration-tests`.

```suggestion
- as: sensor-integration-tests
```

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Comment by gavin-stackrox** [comment]
Excellent. I *hate* all the ~crud~ miscellaneous useful tests in 'non-groovy'. I'd put a `/hold` on this PR if I were you to stop it merging before you are ready on the stackrox side.

**Comment by vikin91** [comment]
Thanks @gavin-stackrox ! Is there a smart way to test the stackrox part without this PR?

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-10.yaml (hunk 1 above)
Comment (vikin91) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-10.yaml]: Done in [523a5f2](https://github.com/openshift/release/pull/40182/commits/523a5f2524aaf51ce7409d88883df750be38e7e0)

**Comment by vikin91** [comment]
I decided to create a dummy script in stackrox (https://github.com/stackrox/stackrox/pull/6399), so that we can merge this easily.
The work for moving the integration tests into the newly prepared target will happen in a followup.

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-ocp-4-12-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-ocp-4-10-ocp-sensor-integration-tests pull-ci-stackrox-stackrox-release-x.y-ocp-4-12-ocp-sensor-integration-tests pull-ci-stackrox-stackrox-release-x.y-ocp-4-13-ocp-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-ocp-4-12-ocp-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-ocp-4-10-ocp-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-ocp-4-12-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-ocp-4-10-sensor-integration-tests pull-ci-stackrox-stackrox-release-x.y-ocp-4-13-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-x.y-ocp-4-13-sensor-integration-tests pull-ci-stackrox-stackrox-master-ocp-4-12-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-nightlies-ocp-4-10-sensor-integration-tests
/pj-rehearse pull-ci-stackrox-stackrox-nightlies-ocp-4-12-sensor-integration-tests
/pj-rehearse pull-ci-stackrox-stackrox-nightlies-ocp-4-13-sensor-integration-tests

**Comment by vikin91** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-nightlies-ocp-4-12-sensor-integration-tests pull-ci-stackrox-stackrox-nightlies-ocp-4-13-sensor-integration-tests

**Comment by gavin-stackrox** [comment]
@vikin91 can you do a follow up PR that uses GCP for the cluster in the 4.12 configs? Similar to the other 4.12 configs.

I'm going to give this a `/lgtm` so that it merges before it gets stale (as openshift/release PRs are prone to). 

/lgtm

**Comment by vikin91** [comment]
@gavin-stackrox could you maybe /lgtm or suggest further changes?

**Pull request merged**